nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -52,8 +52,8 @@
 
       - string:
           name: DISTROS
-          description: "A list of distros to build for. Available options are: bionic, centos8"
-          default: "centos8 bionic"
```

Comment:
Noting that this has been out of sync with `AVAILABLE_DIST` since https://github.com/ceph/ceph-build/pull/2069.